sync/clients/databricks.py

```diff
@@ -48,6 +48,55 @@ def delete_cluster(self, cluster_id: str) -> dict:
             )
         )
 
+    def get_cluster_events(
+        self, cluster_id: str, start_time_ms: int | None = None, end_time_ms: int | None = None
+    ):
```

Comment:
This kinda sets a new precedent for this module. The idea is that client methods are 1 to 1 with API transactions. That said, I think this is the first we needed to page through API responses... an alternative is a private function in awsdatabricks 🤷

I'm ok with it